node-engine/reference/cli/wpe/apps/create.mdx

```diff
@@ -75,6 +79,41 @@ Providing custom `domains` is optional. If you do want to connect a custom domai
 
 **NOTE: We currently only support one custom domain.**
 
+## Create From a zip file
+
+**Note:** The zip app flow only works with wpe CLI version 1.0.0 or greater.
```

Comment:
I'd also add a note about why would anyone like to use this flow... This is an alternative flow for users that use anything else than "cloud" GitHub (so also for users of GitHub Enterprise Server) meanwhile we're working on integrations with remaining version control systems. Just a little background would help I guess :)